packages/nimble-components/src/wafer-map/modules/zoom-handler.ts

```diff
@@ -45,21 +53,74 @@ export class ZoomHandler {
             zoomIdentity.y,
             zoomIdentity.k
         );
-        this.renderingModule.drawWafer(this.zoomTransform.k);
+        this.reScale();
         this.zoomBehavior?.transform(
             select(this.canvas as Element),
             zoomIdentity
         );
     }
 
+    public reScale(): void {
```

Comment:
nit: `reScale` looks funny to me; I'd vote for `rescale`

Reply:
applied